Sekoia.io/trigger_sekoiaio_alert_comment_created.json

```diff
@@ -16,6 +16,14 @@
   "results": {
     "$schema": "http://json-schema.org/draft-07/schema#",
     "properties": {
+      "comment": {
+        "description": "Alert comment (object containing comment uuid, content, author, date and user uuid).",
+        "type": "object"
+      },
```

Comment:
Like the `rule` property, can you describe the subfields of this property?
Can you also add this property to the required ones?